cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -552,7 +554,7 @@ func StartControllers(s *options.CMServer, kubeconfig *restclient.Config, rootCl
 			glog.Fatalf("Failed to get supported resources from server: %v", err)
 		}
 
-		config := restclient.AddUserAgent(kubeconfig, "generic-garbage-collector")
```

Comment:
Compare new comments: two place in the PR don't use the `builder.Client`, but only `builder.Config` which does not set the user-agent.